### PR TITLE
Update DerridasMargins_v1.1_README.txt

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: DerridasMargins_v1.1_README.txt [hunk 1]
```diff
@@ -75,10 +75,6 @@ Contact Information
   * Affiliations: Princeton University, Center for Digital Humanities
   * Email: rebecca.s.koeser@princeton.edu
 
-* Contributor ORCID IDs:
-  * Nick Budak, 
-  * Chad Córdova,
```
Comment (rlskoeser) [inline DerridasMargins_v1.1_README.txt:80]: this looked redundant/extra

Comment (kmcelwee) [reply DerridasMargins_v1.1_README.txt:80]: Correct, certainly a typo, thank you!

File: DerridasMargins_v1.1_README.txt [hunk 2]
```diff
@@ -23,7 +23,7 @@ Dataset Version and Release History
   * Number: 1.1.0
   * Date: 2021-10-14
   * Persistent identifier: <https://doi.org/10.34770/8nc6-4418>
-  * Summary of changes: Added IIIF fields and fixed small errors found in the original dataset. Another CSV containing insertion data was added.
+  * Summary of changes: Added IIIF fields and fixed small errors found in the 1.1 version of the dataset A new CSV containing insertion data was added.
```
Comment (kmcelwee) [inline DerridasMargins_v1.1_README.txt]: 1.0 version? This is 1.1?

Comment (kmcelwee) [reply DerridasMargins_v1.1_README.txt]:
```suggestion
  * Summary of changes: Added IIIF fields and fixed small errors found in the 1.0 version of the dataset. A new CSV containing insertion data was added.

```

**Review by kmcelwee — APPROVED** [review]

**Comment by kmcelwee** [comment]
@rlskoeser Let me know if I should shift Alexander Baron-Raiffe to contributors so that it's in line with the dataspace metadata

**Comment by rlskoeser** [comment]
> @rlskoeser Let me know if I should shift Alexander Baron-Raiffe to contributors so that it's in line with the dataspace metadata

that makes sense to me

**Pull request merged**